MapboxNavigation/NavigationMapView.swift

```diff
@@ -270,6 +274,10 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         installUserCourseView()
     }
     
+    deinit {
+        buildingHighlightPollingTimer?.invalidate()
```

Comment:
The downside to managing the timer in NavigationMapView is that we may need to invalidate it in more places to avoid memory leaks. For one thing, what should happen if client code sets a new route or sets a new style?